src/config/__jest__/Config.test.ts

```diff
@@ -0,0 +1,109 @@
+import Config from '../Config';
+
+jest.mock('fs');
+const randomString = (length: number) =>
+  Math.random()
+    .toString(36)
+    .substring(length);
+
+describe('Default config', () => {
+  const testedConfig = new Config();
+
+  test('Default clientID and token are empty', () => {
+    expect(testedConfig.clientId).toEqual('');
+    expect(testedConfig.token).toEqual('');
+  });
+
+  test('Default language is English', () => {
+    expect(testedConfig.language).toEqual('en');
+  });
+
+  test('Default command prefix is !', () => {
+    expect(testedConfig.prefix).toEqual('!');
+  });
+
+  test('Default whitelisted extensions are mp3 and wav files', () => {
+    expect(testedConfig.acceptedExtensions).toEqual(['.mp3', '.wav']);
+  });
+
+  test('Default maximum filesize is ', () => {
+    expect(testedConfig.maximumFileSize).toEqual(1000000);
+  });
+
+  test('Default setting to delete messages is false', () => {
+    expect(testedConfig.deleteMessages).toEqual(false);
+  });
+
+  test('Default setting to stay in the channel is false', () => {
+    expect(testedConfig.stayInChannel).toEqual(false);
+  });
+
+  test('Default setting to deafen the bot is false', () => {
+    expect(testedConfig.deafen).toEqual(false);
+  });
+
+  test('Default game is not set', () => {
+    expect(testedConfig.game).toEqual('');
+  });
+});
+
+describe('Setting config from Environment Variables', () => {
+  const OLD_ENVIRONMENT_VARIABLES = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
```

Comment:
Not sure we need this call to `resetModules`.

Actually I think we don't need the whole `beforeEach` block here, as `afterEach` should take care of that.

Reply:
You are right, it works without! Thanks!